src/integrationTest/java/org/opensearch/security/privileges/ActionPrivilegesTest.java

```diff
@@ -1071,8 +1071,7 @@ static SecurityDynamicConfiguration<RoleV7> createRoles(int numberOfRoles, int n
     }
 
     static PrivilegesEvaluationContext ctx(String... roles) {
-        User user = new User("test_user");
-        user.addAttributes(ImmutableMap.of("attrs.dept_no", "a11"));
+        User user = new User("test_user").withAttributes(ImmutableMap.of("attrs.dept_no", "a11"));
```

Comment:
Should we consider using the Builder pattern which is pretty prevalently used across OpenSearch?